rules/integrations/aws/lateral_movement_ec2_instance_console_login.toml

```diff
@@ -0,0 +1,81 @@
+[metadata]
+creation_date = "2024/07/24"
+integration = ["aws"]
+maturity = "production"
+updated_date = "2024/07/24"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies a successful console login activity by an EC2 instance profile using an assumed role. This is uncommon behavior and could indicate an attacker using compromised credentials to further exploit an environment. An EC2 instance assumes a role using their EC2 ID as the session name. This rule looks for the pattern "i-" which is the beginning pattern for assumed role sessions started by an EC2 instance and a successful `ConsoleLogin` or `GetSigninToken` API call.
+"""
+false_positives = ["This is very uncommon behavior and should result in minimal false positives, ensure validity of the triggered event and include exceptions where necessary."]
+from = "now-6m"
```

Comment:
Any specific reason for 6m lookback window?

Reply:
Since I left the `from` as default 5 min, this just provides an extra minute detection overlap